### PR TITLE
Fix build with Rust v1.80.0-nightly (2024-05-05).

### PR DESCRIPTION
**Pull request by martijneken**

### CODE REVIEW

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -18,11 +18,11 @@ on:
...
-    - master
+    - main
 
   push:
     branches:
-    - master
+    - main
 
   schedule:
     - cron: '0 0 * * *'
```
Comment (PiotrSikora) [inline .github/workflows/rust.yml:28]: Could you remove those changes, since they'll be merged in #234? Thanks!

**Review by PiotrSikora — DISMISSED** [review]

File: .github/workflows/rust.yml (hunk 1 above)
Comment (martijneken) [reply .github/workflows/rust.yml:28]: Yep included these to get CI checks. Will remove when #234 is merged.

**Comment by martijneken** [comment]
PTAL, https://github.com/proxy-wasm/proxy-wasm-rust-sdk/pull/236 fixes the last CI failure

**Review by PiotrSikora — APPROVED** [review]
Thanks!

**Pull request merged**